src/controllers/users.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const userService = require('../services/users.js');
+
+const getAll = (req, res) => {
+  const users = userService.getAll();
+
+  res.send(users);
+};
+
+const getOne = (req, res) => {
+  const { userId } = req.params;
+  const foundUser = userService.getUserById(userId);
+
+  if (!foundUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundUser);
+};
+
+const add = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newUser = userService.create(name);
+
+  res.statusCode = 201;
+  res.send(newUser);
+};
+
+const remove = (req, res) => {
+  const { userId } = req.params;
+  const foundUser = userService.getUserById(userId);
+
+  if (!foundUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  userService.remove(userId);
+  res.sendStatus(204);
+};
+
+const update = (req, res) => {
+  const { userId } = req.params;
+  const foundUser = userService.getUserById(userId);
+
+  if (!foundUser) {
+    res.sendStatus(404);
```

Comment:
The same
Make it in all cases
```suggestion
    res.sendStatus(404);
    
    return;
```